guides/release/templates/binding-element-attributes.md

```diff
@@ -38,39 +38,41 @@ If `isAdministrator` is `false`, Handlebars will produce the following:
 <input type="checkbox">
 ```
 
-### Adding Other Attributes (Including Data Attributes)
+### Adding Attributes to Component Invocations
 
-By default, components only accept a limited number of HTML attributes.
-This means that some uncommon but perfectly valid attributes, such as `lang` or
-custom `data-*` attributes must be specifically enabled. For example, this template:
+Components in Ember can also accept attributes to be places on an element
+which they render. A common use for this might be `lang` or `data-*`
+attributes. For example, this template passes those two attributes to
+a link component:
 
 ```handlebars
-{{#link-to "photos" data-toggle="dropdown" lang="es"}}Fotos{{/link-to}}
+<LinkTo @route="photos" data-toggle="dropdown" lang="es">
+  Fotos
+</LinkTo>
 ```
 
-Will render the following HTML:
+Which will render render the following HTML:
 
 ```html
-<a id="ember239" class="ember-view" href="#/photos">Fotos</a>
+<a href="#/photos" data-toggle="dropdown" lang="es" id="ember239" class="ember-view">
```

Comment:
Seems worth simplifying down to what we'd like produce here?
```suggestion
<a href="/photos" data-toggle="dropdown" lang="es" id="ember239" class="ember-view">
```

Reply:
Cool, confirmed the non-hash version is actually what we produce out of the box.